index.js

```diff
@@ -1,22 +1,27 @@
 var LESSCompiler = require('broccoli-less-single');
+var path         = require('path');
+var merge        = require('lodash-node/modern/objects/merge');
+var mergeTrees   = require('broccoli-merge-trees');
 
 function LESSPlugin(options) {
   this.name = 'ember-cli-less';
   this.ext = 'less';
-  options = options || {};
-  options.inputFile = options.inputFile || 'app.less';
-  options.outputFile = options.outputFile || 'app.css';
   this.options = options;
 }
 
-LESSPlugin.prototype.toTree = function(tree, inputPath, outputPath) {
-  var trees = [tree];
-  if (this.options.paths) {
-    trees = trees.concat(this.options.paths);
-  }
-  inputPath += '/' + this.options.inputFile;
-  outputPath += '/' + this.options.outputFile;
-  return new LESSCompiler(trees, inputPath, outputPath, this.options);
+LESSPlugin.prototype.toTree = function(tree, inputPath, outputPath, options) {
+  options = merge({}, this.options, options);
+  var _this = this,
+      paths = options.outputPaths;
+
+  var trees = Object.keys(paths).map(function (file) {
```

Comment:
I think you're confusing ember.js and ember-cli?  This has to do with the ember-cli build tool, where outputPaths has always been present since version 0.0.45